plugins/trivy/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 0.23.3
```

Comment:
The new version should probably be `0.23.0` right?

Reply:
Doe - thank you!